packages/cli/src/ceramic-daemon.ts

```diff
@@ -106,6 +106,9 @@ export function makeCeramicConfig(opts: DaemonConfig): CeramicConfig {
     )
   } // else do nothing, no metrics are configured.
 
+  const disablePeerDataSync =
+    opts.ipfs.disablePeerDataSync || process.env.CERAMIC_DISABLE_IPFS_PEER_DATA_SYNC == 'true'
```

Comment:
all the other env variable overrides are in ./packages/cli/src/ceramic-cli-utils.ts in CreateDaemon i think we should put this there

Reply:
good call, done